packages/kit/src/runtime/server/page/render.js

```diff
@@ -173,7 +171,12 @@ export async function render_response({
 	].join('\n\n\t\t');
 
 	const body = options.amp
-		? rendered.html
+		? `${rendered.html}
+			${
+				options.service_worker
+					? `<amp-install-serviceworker src="${options.service_worker}" layout="nodisplay"></amp-install-serviceworker>`
+					: ''
+			}`
 		: `${rendered.html}
```

Comment:
I think we can refactor this as:

```js
let body = rendered.html

if (options.amp) {
  if (options.service_worker) {
    body += ''
  }
} else {
  body += ''
}
```

to make it a bit readable.

Reply:
I agree with your suggestion & have modified the code, thanks 👍🏼